### ISSUE TITLE
Unable to select the first option in the date filter dropdown

### ISSUE THREAD

**Issue by srish** [wikimedia/WikiContrib#41]
Dropdown filter is working fine for all dates but not "September 2019". It is the first one in the list, and I am not able to select it. 

<img width="391" alt="Screen Shot 2019-09-26 at 11 54 26 PM" src="https://user-images.githubusercontent.com/288353/65748701-646dcd80-e0b9-11e9-82c2-9f33c2837b3a.png">

**Comment by srish** [comment]
Same for the "Time Range" filter as well. I'm not able to select the first option in it.

**Comment by Ammarpad** [comment]
Hi @srish, I cannot reproduce this bug. I can select both current month (October) and any other. Likewise I can select any time range.

**Comment by srish** [comment]
@Ammarpad That's interesting! I'm still seeing this on my end. Which browser are you on, and would it be possible for you to paste your console.log() output here when you select `October 2019` in the date dropdown and `Last 30 days` in the Time range dropdown? Thanks!

**Comment by Ammarpad** [comment]
I am using 'Google Chrome 77.0.3865.90 (Official Build) (64-bit)' on macOS. 

Console output here:
[tools.wmflabs.org-1570591856062.log](https://github.com/wikimedia/WikiContrib/files/3705519/tools.wmflabs.org-1570591856062.log)

I also added a 1-minute screencast: [here at imgur.com](https://imgur.com/Izqvpzu)

**Comment by srish** [comment]
@Ammarpad WOW! I've the same setting as well and it is not working for me. Also, I tried Safari and Firefox, but the problem persists. We need a third person now to help verify :)

**Comment by rammanoj** [comment]
@srish are there any console errors when you try to select the first element of the list?

**Comment by rammanoj** [comment]
> I am using 'Google Chrome 77.0.3865.90 (Official Build) (64-bit)' on macOS.
> 
> Console output here:
> [tools.wmflabs.org-1570591856062.log](https://github.com/wikimedia/WikiContrib/files/3705519/tools.wmflabs.org-1570591856062.log)
> 
> I also added a 1-minute screencast [here at imgur.com](https://imgur.com/Izqvpzu). (Note pointer movement)

@Ammarpad Thanks for pointing this out! I guess this creates a new issue!

**Comment by srish** [comment]
I figured the reason why this is happening. For the end date being November 1st (default setting) the end month should be October, but the output I get for https://github.com/wikimedia/WikiContrib/blob/master/frontend/WikiContrib-Frontend/src/result.js#L631 is in PST (my current timezone) i.e. September 30th. This is also the reason why I see a mismatch in the plotted contributions that I reported in https://github.com/wikimedia/WikiContrib/issues/40.
